GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/DehydrateTests.cs

```diff
@@ -65,10 +65,7 @@ public void DehydrateShouldBackupFiles()
 
             // .git folder items
             string gitFolder = Path.Combine(backupFolderItems[0], ".git");
-            this.DirectoryShouldContain(gitFolder, "index", "info");
-
-            string gitInfoFolder = Path.Combine(gitFolder, "info");
-            this.DirectoryShouldContain(gitInfoFolder, "sparse-checkout");

```

Comment:
Unrelated to your changes, does the sparse-checkout do anything when the ModifiedPaths.dat file is being used?

Reply:
The file is not doing anything when using the virtual projection feature instead but git is still using some of the sparse-checkout code internally.  I think for git it is either virtual projection or sparse-checkout since the virtual projection turns on sparse-checkout internally. @benpeart is that right?

Reply:
Correct.  The virtual projection uses the internal skip-worktree bit (and excludes) logic but just sets the bits using different logic (ie it doesn't use the sparse-checkout and always-excludes files, but the virtual projection file instead).